src/grpc_generated/go/grpc_simple_client.go

```diff
@@ -35,7 +35,7 @@ import (
 	"log"
 	"time"
 
-	triton "nvidia_inferenceserver"
+	triton "github.com/triton-inference-server/client/src/grpc_generated/go/grpc-client"
```

Comment:
> Hmm realised that in order to export the go package we need to start using tags based upon semver, would that make sense in relation to the current release structure of the client?

Can you elaborate on this? Triton does have a semver release tag on the server repo, currently 2.24.0 at time of writing: https://github.com/triton-inference-server/server/releases/tag/v2.24.0

What are the implications of exporting the package vs just having a local example like we currently do?

Reply:
1) For Go (as I understand) you need to have a tagged version(semver) in order for other developers to be able to import the package(go modules), more info [here](https://go.dev/doc/modules/release-workflow#unpublished)
2) The implication of this is that any one that want to use the go grpc client will have to regenerate and put it local to their own code. 

> This is not currently meant to be a full fledged Go-GRPC-Triton package, just a simple example for reference.

Maybe then the goal is actually not to make it possible to import for other people? I would at least appreciate to have it available as a lib to not have to generate it myself. However to actually do this in a good way the client should be regenerated when the protos change preferable using (CI/CD).

Reply:
> Maybe then the goal is actually not to make it possible to import for other people? I would at least appreciate to have it available as a lib to not have to generate it myself. However to actually do this in a good way the client should be regenerated when the protos change preferable using (CI/CD).

Yeah this was my concern - I don't think maintaining a public package for the Go client is in scope right now. I think we would need to see more demand/support to consider maintaining this in the future. For the sake of this example, I think it is acceptable to have the user generate the stubs on-demand at this time. This is aligned with how our Java and Javascript grpc-generated examples work today as well. For other projects or more complicated examples, the generated stubs can be copied out / re-used as needed.

Reply:
@NikeNano Given that we don't want to maintain an public package for the go client at this time as mentioned above, is this import still valid?
```
	triton "github.com/triton-inference-server/client/src/grpc_generated/go/grpc-client"
```

Reply:
Sorry for the late replay, have been over loaded last couple of days. Yes it is still valid.

Reply:
It will refer to the local path in this case.